easy-setup.sh

```diff
@@ -0,0 +1,82 @@
+OS_BASE="$(uname)"
+BUILD_MAC=$"Darwin"
+BUILD_LIN=$"Linux"
+BUILD_WIN="Windows"
+KERALARESCUE_REPO_LINK="https://github.com/IEEEKeralaSection/rescuekerala.git"
+
+# Commands for Apple Mac
+if [ $OS_BASE == $BUILD_MAC ] 
+then
+# Install Git and the repo for Kerala Rescue
+brew install git
+# ======= Uncomment the following line if you do not have the repository downloaded
```

Comment:
The docker installation is missing `brew cask install docker`.

Reply:
Docker is not mandatory, as per my understanding. The docker might repeat the same stuffs again.